deployment/terraform/nixops.tf

```diff
@@ -19,8 +19,8 @@ resource "aws_security_group" "nixops" {
   }
 
   ingress {
-    from_port   = "3000"
-    to_port     = "3000"
+    from_port   = "80"
+    to_port     = "80"
```

Comment:
Should some of these ports in various places be variables?

Reply:
generally not, it doesn't make much sense to override them since everyone can create their own independent environment.

Reply:
Yeah but for example I have no idea whether these have to be the same as the other port 80s or not.

Reply:
So it's mainly for legibility.

Reply:
I can add comments or local variables (effectively named constants)?

Reply:
Yeah named constants is good.